src/Sql/Sql/Az.Sql.psd1

```diff
@@ -265,7 +265,9 @@ AliasesToExport = 'Get-AzSqlDatabaseServerAuditingPolicy',
                'Get-AzSqlDatabaseThreatDetectionPolicy', 
                'Set-AzSqlDatabaseThreatDetectionPolicy', 
                'Remove-AzSqlDatabaseThreatDetectionPolicy', 
-               'Set-AzSqlInstanceTDEProtector', 'Get-AzSqlInstanceTDEProtector'
+               'Set-AzSqlInstanceTDEProtector', 'Get-AzSqlInstanceTDEProtector',
+			   'Invoke-AzureSqlDatabaseFailover',
```

Comment:
@emgu-ms would you mind fixing the identation here? It looks like it's an issue with tabs vs. spaces 😁